layout/widgets/login.php

```diff
@@ -10,21 +10,23 @@
 			<li>
 				Password: <br>
 				<input type="password" name="password">
-			</li>
+			</li><?php if ($config['use_token'] == true) { ?>
 			<li>
 				Token: <br>
 				<input type="password" name="authcode">
```

Comment:
This input field is actually for Two-Factor Authentication
https://github.com/Znote/ZnoteAAC/commit/c3c236e13ed3002ecac025aeed2e9f9427e95625#diff-36f5eddf48f10c8608c7fcca1fa1a1d8
And is the 6 digit code you get from your mobile authenticator. Not the same as the token system. 
I see that I called them both token, so I understand the confusion. 

Use ```php$config['twoFactorAuthenticator']``` instead.